biofefi/options/plotting.py

```diff
@@ -1,11 +1,14 @@
 from dataclasses import dataclass
 
-
+"""In this file, we define the PlottingOptions dataclass, which is used to store the plotting 
+preferences, defined in the UI for the plots generated by the BioFEFI package. All plots
+will then have the same style, such as font size, font family, colour scheme, etc. """
 @dataclass
 class PlottingOptions:
     plot_axis_font_size: int
     plot_axis_tick_size: int
     plot_colour_scheme: str
+    plot_colour_map: str
```

Comment:
Since this is a new key, experiments made on older versions of BioFEFI won't be compatible unless you give this a default value. The way if experiments are loaded that didn't have the `plot_colour_map` key, BioFEFI can just assign the default and not break. 

To do this, put `plot_colour_map` at the bottom of the class and do `plot_colour_map: str = <insert value>`